app/jobs/record_claim_event_job.rb

```diff
@@ -0,0 +1,6 @@
+class RecordClaimEventJob < ApplicationJob
+  def perform(claim_ids, event_type, date_field)
```

Comment:
The parameter is called `date_field` here but `performed_at_method` in the `Claim::GeckoboardEvent` class. Worth making them consistent? Also in the `body_matching_geckoboard_data_for_claims` method.